File: Visual Basic-Ödev 2/Visual Basic-Ödev 2/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        ListBox1.Items.Add(TextBox1.Text + " " + TextBox2.Text + " " + MaskedTextBox1.Text + " " + MaskedTextBox2.Text + " " + ComboBox1.Text + " " + DateTimePicker1.Text)
    End Sub
End Class

File: C#-Do-While Döngüsü/C#-Do-While Döngüsü/Form1.cs
namespace C__Do_While_Döngüsü
{
	public partial class Form1 : Form
	{
		public Form1()
		{
			InitializeComponent();
		}

		private void button1_Click(object sender, EventArgs e)
		{
			int i = 1;
			do
			{
				listBox1.Items.Add("Merhaba Niğde");
				i++;
			}
			while (i <= 10);
		}
	}
}